HttpSignature.md

```diff
@@ -4,25 +4,23 @@
 The work is based on [draft-cavage-http-signature-12](https://tools.ietf.org/html/draft-cavage-http-signatures-12), which evolved and gained adoption since 2013, being tested by a [large number of implementations](https://github.com/w3c-dvcg/http-signatures/issues/1), and this is set to grow by being taken up by the IETF.
 
 HTTP Signature has the advantages of being very simple and being specified directly at the HTTP layer, bypassing the problem of client authentication at the TLS layer.
-(Note: all communication here is assumed to run over TLS.) The protocol allows a client to authenticate by signing any of several HTTP headers with any one of its private keys.
+(Note: all communication here is assumed to run over TLS.) 
+
+The protocol allows a client to authenticate by signing any of several HTTP headers with any one of its private keys.
 In order for the server to verify this signature, it needs to know the matching public key.
 This information must be transmitted by the client, in the form of an opaque string known as a `keyId` (see [§2.1.1 keyId](https://tools.ietf.org/html/draft-cavage-http-signatures-11#section-2.1.1)).
 This string must enable the server to look up the key; how this look-up is done is not specified by the protocol.
 
 This `Http-Sig` protocol extension allows the `keyId` to be interpreted as a URL.
-The proposal here is to use an `https` URL identifier ending with a fragment for the `keyId`.
-This proposal extension is compatible with the keyId using other URI schemes such as [DID](https://www.w3.org/TR/did-core/)s. 
+The main use case is to allow the use of an `https` URL identifier ending with a fragment for the `keyId`, but it would also allow other URI schemes such as [DID](https://www.w3.org/TR/did-core/)s. 
 
- In order for a server to discover the key, it can fetch the `keyId Document`, whose URL is given by the `keyId` URL without the fragment identifier (see [§3 of RFC 3986: Uniform Resource Identifier (URI): Generic Syntax](https://tools.ietf.org/html/rfc3986?#section-3)).  
+By allowing URLs to be used in the `keyId` field,  we make it possible for the server to discover the key by fetching the `keyId Document`, whose URL is given by the [resolved](https://tools.ietf.org/html/rfc3986#section-5) `keyId` URL without the fragment identifier (see [§3 of RFC 3986: Uniform Resource Identifier (URI): Generic Syntax](https://tools.ietf.org/html/rfc3986?#section-3)).  
 
 ## Extending `HTTP-Sig` with URLs
 
-Here we consider the minimum extension of `HTTP-Sig` with
-`keyId`s that are URLs.
-We point to the advantages of this
-in terms of enabling client creation, editing, and deleting of keys.
-We then show how this ties into the larger Access
-Control Protocol used by Solid.
+Here we consider the minimum extension of `HTTP-Sig` with `keyId`s that are URLs.
+We point to the advantages of this in terms of enabling client creation, editing and deleting of keys.
```

Comment:
```suggestion
We point to the advantages of this in terms of enabling client creation, and of the editing and deleting of keys.
```

Reply:
I removed that text and instead moved the text about GET, POST, DELETE up into this section.